open_samus_returns_rando/files/schema.json

```diff
@@ -335,7 +335,46 @@
                 "ITEM_METROID_TOTAL_COUNT",
                 "ITEM_ADN",
                 "ITEM_RANDO_LOCKED_SUPERS",
-                "ITEM_RANDO_LOCKED_PBS"
+                "ITEM_RANDO_LOCKED_PBS",
+                "ITEM_RANDO_DNA_1",
+                "ITEM_RANDO_DNA_2",
+                "ITEM_RANDO_DNA_3",
+                "ITEM_RANDO_DNA_4",
+                "ITEM_RANDO_DNA_5",
+                "ITEM_RANDO_DNA_6",
+                "ITEM_RANDO_DNA_7",
+                "ITEM_RANDO_DNA_8",
+                "ITEM_RANDO_DNA_9",
+                "ITEM_RANDO_DNA_10",
+                "ITEM_RANDO_DNA_11",
+                "ITEM_RANDO_DNA_12",
+                "ITEM_RANDO_DNA_13",
+                "ITEM_RANDO_DNA_14",
+                "ITEM_RANDO_DNA_15",
+                "ITEM_RANDO_DNA_16",
+                "ITEM_RANDO_DNA_17",
+                "ITEM_RANDO_DNA_18",
+                "ITEM_RANDO_DNA_19",
+                "ITEM_RANDO_DNA_20",
+                "ITEM_RANDO_DNA_21",
+                "ITEM_RANDO_DNA_22",
+                "ITEM_RANDO_DNA_23",
+                "ITEM_RANDO_DNA_24",
+                "ITEM_RANDO_DNA_25",
+                "ITEM_RANDO_DNA_26",
+                "ITEM_RANDO_DNA_27",
+                "ITEM_RANDO_DNA_28",
+                "ITEM_RANDO_DNA_29",
+                "ITEM_RANDO_DNA_30",
+                "ITEM_RANDO_DNA_31",
+                "ITEM_RANDO_DNA_32",
+                "ITEM_RANDO_DNA_33",
+                "ITEM_RANDO_DNA_34",
+                "ITEM_RANDO_DNA_35",
+                "ITEM_RANDO_DNA_36",
+                "ITEM_RANDO_DNA_37",
+                "ITEM_RANDO_DNA_38",
+                "ITEM_RANDO_DNA_39"
```

Comment:
What happens if you don't start with any dna or only some of it? Will the schema throw an error?

Reply:
No, it will not throw an error. This is just the enumeration of what items exists but none of these are required.

Reply:
Ah right.